### PR TITLE
New blog post: 2021 Recap

### PR DESCRIPTION
**Pull request by ByteHamster**

### CODE REVIEW

File: _blog/2021-12-29-2021 Recap.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Developing a [home screen](https://github.com/AntennaPod/AntennaPod/issues/3952) with a quick overview over the app.
+- Adding [another episode state "ignored"](https://github.com/AntennaPod/AntennaPod/issues/5237), which you can use if you are not interested in an episode, so you do not have to mark episodes as played that are not actually played.
+- Renaming the "new" screen to ["inbox"](https://github.com/AntennaPod/AntennaPod/issues/5267) and adding it directly to the side drawer.
+
+The Hackathon was very productive, and will explore in which form we can continue such live developer interaction in 2022.
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:28]:
```suggestion
The Hackathon was very productive, and we will explore in which form we can continue such live contributor interaction in 2022.
```

File: _blog/2021-12-29-2021 Recap.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+date: "2021-12-29 11:00:00"
+image: "2021/recap.png"
+author: bytehamster
+layout: blog
+---
+
+Looking back at 2021, a lot happened in the AntennaPod community. In this blog post, we want to summarize the most important milestones.
+
+# App development
+**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free-time. To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:13]:
```suggestion
**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free time.

To give everyone access to the app improvements, we released 9 new updates, of which 3 contained major new features.
```
(There currently is no white line, and in retrospect this last sentence better fits with the next bit.)

File: _blog/2021-12-29-2021 Recap.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+For the beginning of 2022, we have already developed a release that is nearly ready. The new release of AntennaPod - version 2.5 - will add support for synchronizing your subscriptions with your own Nextcloud instance. That makes it easier than ever to enjoy sync while still keeping your listening habits private!
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:14]:
```suggestion
For the beginning of 2022, we have already developed a release that is nearly ready. This new release of AntennaPod -- version 2.5 -- will add support for synchronizing your subscriptions with your own Nextcloud instance. That makes it easier than ever to enjoy sync while still keeping your listening habits private!
```

File: _blog/2021-12-29-2021 Recap.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# Donations
+After a long period of not accepting donations and users asked for it, in **March** this year we decided that we will accept donations. Active community members [discussed](https://forum.antennapod.org/t/should-antennapod-begin-accepting-donations/249/29) what might and should not be paid for with the money, and finally developed [spending guidelines](https://forum.antennapod.org/t/donation-expenses-policy/272/2). In nine months we already received € 1263.75. The first and only spending this year was € 21.60 for a Google Play Console account (to better manage releases). We hope we can put more of the money to good use going forward.
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:17]:
```suggestion
After a long period of not accepting donations and users asked for it, in **March** this year we decided that we will accept donations. Active community members [discussed](https://forum.antennapod.org/t/should-antennapod-begin-accepting-donations/249/29) what might and should not be paid for with the money, and finally developed [spending guidelines](https://forum.antennapod.org/t/donation-expenses-policy/272/2). In nine months we already received € 1 263.75. The first and only spending this year was € 21.60 for a Google Play Console account (to better manage releases). We hope we can put more of the money to good use going forward.
```

File: _blog/2021-12-29-2021 Recap.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+On January 1st, the great forum software (Discourse) [will](https://forum.antennapod.org/t/2021-the-year-in-review) automatically publish a full overview of the forum activity. (So check in on our forum in a few days if you're reading this in 2021.)
+
+# See you again next year
+We hope that you had a good year, despite the global pandemic. Having to stay at home is a good time to listen to podcasts, after all 😉 Now, we wish you a good transition into the year 2022, and we will see you again next year!
+
+PS. Don't hesitate to create a post on our forum if you have any comments about this blog post. As said, it also is a good place to discuss new features and help shape the future of AntennaPod. And if you want to help to make AntennaPod even greater, have a look at our website where you can [contribute](/contribute) in various ways such as translating or coding.
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:37]:
```suggestion
PS. Don't hesitate to create a post on our forum if you have any comments about this blog post. As said, it also is a good place to discuss new features and help shape the future of AntennaPod. And if you want to help to make AntennaPod even greater, have a look at our website where we introduce the various ways in which you can [contribute](/contribute) to the project, such as translating or coding.
```

**Review by keunes — COMMENTED** [review]
A few more small change proposals. It'd be great if we could publish tomorrow. Guess it's fine if we don't have updated commit numbers – maybe we can simply change it to 'well over…'

File: _blog/2021-12-29-2021 Recap.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+---
+title: 2021 Recap
+excerpt: What happened in the AntennaPod project in 2021
+date: "2021-12-29 11:00:00"
```
Comment (keunes) [inline _blog/2021-12-29-2021 Recap.md:4]:
```suggestion
date: "2021-12-30 11:00:00"
```

File: _blog/2021-12-30-2021 Recap.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+date: "2021-12-30 11:00:00"
+image: "2021/recap.png"
+author: bytehamster
+layout: blog
+---
+
+Looking back at 2021, a lot happened in the AntennaPod community. In this blog post, we want to summarize the most important milestones.
+
+# App development
+**Over the course of 2021**, 48 contributors helped improve AntennaPod. In total, they made no less than 482 changes (commits) to the source code, touching about 29 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free time.
```
Comment (ByteHamster) [inline _blog/2021-12-30-2021 Recap.md:13]:
```suggestion
**Over the course of 2021**, 51 contributors helped improve AntennaPod. In total, they made no less than 505 changes (commits) to the source code, touching about 30 000 lines of code. This makes AntennaPod a pretty active project, even though it is run completely by volunteers in their free time.
```

**Pull request merged**